packages/monaco/src/EditorHooks.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
```

Comment:
we kebab case our file names but i'd argue you can just put this in the index file unless you see this growing

Reply:
oh i see its a separate entry point for importing